apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -5047,6 +5048,8 @@ suite('conversation.js >', function() {
         content: body,
         serviceId: 0
       });
+      this.sinon.stub(Compose, 'clear');
+      sinon.assert.notCalled(Compose.clear);
```

Comment:
there is something you don't understand properly here. The "stub" needs to be setup before you do the action. You need that the stub is set up before we simulate the click, so that we can test that the stub is called/not called as a result of the click action.
Here you're not testing anything.

So you can set up the stub at the start of the test function, for example.